### DISCUSSION TITLE
Adding references using PMID

### DISCUSSION THREAD

**Discussion by choxos** [quarto-dev/quarto-cli#509] [Q&A]
Hi,

It seems some of the PubMed references could not be found by Insert Citation tool. For instance: https://pubmed.ncbi.nlm.nih.gov/20930260/. It would be great if you added inserting citations by PMID (just like Zotero).

Ahmad

**Comment by dragonstyle** [comment]
That definitely should work, but I can reproduce that it doesn't. I can see an error being logged internally, so will investigate.
